README.md

```diff
@@ -241,6 +251,35 @@ export const errorResponseSchemas = [
 ]
 ```
 
+Implementing custom extractor example.
+```typescript
+/**
+ * Custom extractor
+ * @param endpointMiddleware object containing all necessary information about actual middleware 
+ * or undefined in case the middleware is not used in a given endpoint.
+ * @param configMiddleware config information of a middleware wich should be processed
+ * @return { string } middleware's description
+ * */
+export const myExtractor = (endpointMiddleware: IEndpointMiddleware | undefined, configMiddleware: ISwaggerMiddlewareConfig): string => {
```

Comment:
@kot-1999 would be better just use refence on defaultExtractor in utils

Reply:
@LubomirIgonda1  It's possible, but I managed by the whole structure of this Readme.md and I think it's better to see the implementation at once